tests/serialization/binary_compatibility/README.md

```diff
@@ -0,0 +1,226 @@
+The serialization binary compatibility file is autogenerated using
```

Comment:
Nice documentation! It'll be certain valuable in future when the test needs to be updated.